components/script/dom/element.rs

```diff
@@ -692,6 +693,28 @@ impl LayoutElementHelpers for LayoutJS<Element> {
         }
     }
 
+    #[allow(unsafe_code)]
+    // https://html.spec.whatwg.org/multipage/#language
+    fn language(&self) -> Atom {
```

Comment:
Probably call this `get_lang_for_layout` or similar so it's more obvious that you're calling the correct method below.